CBLClient/Apps/CBLTestServer-C/build_scripts/build_cross.py

```diff
@@ -159,6 +167,12 @@ def check_sysroot(name: str):
         tar.extractall(DOWNLOAD_DIR)
     os.remove(zip_filename)
 
+    vector_search_zip_name = f'couchbase-lite-vector-search-1.0.0-42-linux-x86_64.zip'
```

Comment:
Hardcoding VectorSearch version here, would it be worth leaving a comment in to suggest updating with future versions.

Reply:
Thanks for picking that up, fixed! (tested with building and then running the vector. search test)